pkg/minikube/audit/audit.go

```diff
@@ -58,7 +58,7 @@ func args() string {
 
 // Log details about the executed command.
 func LogCommandStart() (string, error) {
-	if len(os.Args) < 2 || !shouldLog() {
+	if !shouldLog() {
```

Comment:
For context, the arg length check was previously moved into `shouldLog()` this is a leftover check.